src/Surfnet/AzureMfa/Infrastructure/Controller/DefaultController.php

```diff
@@ -176,7 +176,7 @@ public function acsAction(Request $request)
                 $this->authenticationService->authenticate();
             }
         } catch (Exception $e) {
-            $this->logger->error('The authentication or registration failed. Rejecting the Azure MFA response.');
+            $this->logger->error('The authentication or registration failed. Rejecting the Azure MFA response. '.$e->getMessage());
```

Comment:
Adding the exception message might be very usefull. Please concider encapsulating it in double quotes. And may I suggest using  the `sprintf` function to jam the message into the existing. Example:

```
$this->logger->error(
    sprintf(
         'original message with exception: "%s"',
         $e->getMessage()
    )
);
```

Reply:
Thank you; I've updated according to your example.